brigade-worker/src/k8s.ts

```diff
@@ -319,6 +320,11 @@ export class JobRunner implements jobs.JobRunner {
         // For environmental variables that are directly references,
         // add the reference to the env var list.
 
+        if (val.secretKeyRef != null && !allowSecretKeyRef) {
+         // allowSecretKeyRef is not to true so disallow setting secrets in the environment
```

Comment:
Thoughts on logging this message to user, on the off-chance `allowSecretKeyRef` is currently `false` but not known to current user expecting secretKeyRefs in env?

Reply:
Yup logging to the user makes sense, I'll get that done tomorrow.

Reply:
dome